dev/com.ibm.ws.webcontainer.cors/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -15,19 +15,20 @@
 #NLS_MESSAGEFORMAT_NONE
 
 cors=Cross-Origin Resource Sharing
-cors.desc=Refers to a cross-domain security procedure for JavaScript-based clients.
+cors.desc=Configuration for the Cross-Origin Resource Sharing (CORS).
 
 domain=Domain name
 domain.desc=The domain name represents the URL being setup with these CORS settings.
 
 allowedOrigins=Allowed origins
-allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain.
+allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain. 
+allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain. If this is set to "*", it indicates the resource is publicly accessible, with no access control checks. Also, the value of "null" indicates, the domain must not be accessible.  
 
 allowedMethods=Allowed methods
 allowedMethods.desc=A comma-separated list of HTTP methods allowed to be used by the origin domain when making requests to the configured domain.
 
 allowedHeaders=Allowed headers
-allowedHeaders.desc=A comma-separated list of HTTP headers allowed to be used by the origin domain when making requests to the configured domain.
+allowedHeaders.desc=A comma-separated list of HTTP headers allowed to be used by the origin domain when making requests to the configured domain. If this is set to "*", it indicates the request can include any HTTP headers.
```

Comment:
allowedHeaders.desc=A comma-separated list of HTTP headers **that are** allowed to be used by the origin domain when making requests to the configured domain. If this is set to "*", it indicates the request can include any HTTP headers.